doc/developer/standards_and_practices.rst

```diff
@@ -298,17 +298,23 @@ Input files
 ARMI developers **shall** use one of the following well-defined, Python-supported, input file formats.
 
 .json
-    JSON files are used for a variety of data-object representations. There are some limitations of JSON, in that it
-    does not easily support comments. JSON is also very strict.
+    JSON files are used for a variety of data-object representations. There are some limitations of
+    JSON, in that it does not easily support comments. JSON is also very strict.
 
 .yaml
     YAML files are like JSON files but can have comments in them.
 
 General do's and don'ts
 =======================
 
-do not use ``print``
+Do not use ``print``
     ARMI code should not use the ``print`` function; use one of the methods within ``armi.runLog``.
 
 Do not add new ``TODO`` statements in your commits and PRs.
-    If your new ``TODO`` statement is important, it should be a GitHub Issue. Yes, we have existing ``TODO`` statements in the code, those are relic and need to be removed. Also, never mark the code with ``FIXME`` or ``XXX```; open a ticket.
+    If your new ``TODO`` statement is important, it should be a GitHub Issue. Yes, we have existing
+    ``TODO`` statements in the code, those are relic and need to be removed. Similarly, never mark
+    the code with ``FIXME`` or ``XXX```; open a ticket.
+
+Do not link GitHub tickets or PRs in code.
+    The idea in ARMI is that either something is worth documenting well in a docstring, or the docs,
+    or it is not. And just linking a ticket or PR in a docstring is not helpful.
```

Comment:
I mean, my notion goes like this:

1. We have managed for a decade without doing this yet.
2. I have seen projects that do this, and inevitably, their docstrings and internal docs suffer hard.
3. I personally, me, don't want to have to go do homework when I'm reading the code. It takes what should be reading a one-sentence docstring to reading a 4-page story.
4. I don't think "background" and "history" help people understand code. If you can't say "This works like this, because of this" then just giving an infinite amount of context and history and background is just telling people you can't document the feature clearly, and are giving up.

BUT I will admit there is a lot of personal bias above, so let me think about this some more.

Reply:
Also, it occurs to me that GitHub is the third git platform/tool this project has been on.

Pointing to a GitHub ticket isn't particularly useful if we've moved on to GitLab or whatever the new hotness is in 6 years.

Reply:
All great points. I don't have any good rebuttals. And we should definitely encourage any documentation to stand on its own